pallet-chainlink-feed/src/tests.rs

```diff
@@ -603,6 +604,24 @@ fn update_future_rounds_should_work() {
 	});
 }
 
+#[test]
+fn force_admin_transfer_should_work() {
+	new_test_ext().execute_with(|| {
+		let current_admin = ChainlinkFeed::pallet_admin();
+		let new_admin = 42;
```

Comment:
@mattsse isn't this integer a big weird for admin address? Can we change it to something more appropriate?

Reply:
This is according to the configured `AccountId` type in the mock runtime.
It is quite common to use `type AccountId = u128` for the mock scenarios. In the `substrate-node-example` this would be `MultiAddress` of course.